tests/foreman/ui/test_contentview.py

```diff
@@ -3430,3 +3430,367 @@ def test_positive_promote_CV_with_custom_user_role_and_filters(self):
             self.content_views.promote(cv_name, 'Version 1', env_name)
             self.assertIsNotNone(self.content_views.wait_until_element
                                  (common_locators['alert.success_sub_form']))
+
+    @stubbed()
+    @run_only_on('sat')
+    @tier2
+    def test_positive_remove_cv_version_from_default_env(self):
+        """Remove content view version from Library environment
+
+        @id: 43c83c15-c883-45a7-be05-d9b26da99e3c
+
+        @Steps:
+
+        1. Create a content view
+
+        2. Add a yum repo to it
+
+        3. Publish content view
+
+        4. remove the published version from Library environment
+
+        @Assert: content view version is removed from Library environment
+
+        @caseautomation: notautomated
+
+        @CaseLevel: Integration
+        """
+
+    @stubbed()
+    @run_only_on('sat')
+    @tier2
+    def test_positive_remove_renamed_cv_version_from_default_env(self):
+        """Remove version of renamed content view from Library environment
+
+        @id: bd5ca409-f3ab-43b5-bb63-3b747aa75506
+
+        @Steps:
+
+        1. Create a content view
+
+        2. Add a yum repo to the content view
+
+        3. Publish the content view
+
+        4. Rename the content view
+
+        5. remove the published version from Library environment
+
+        @Assert: content view version is removed from Library environment
+
+        @caseautomation: notautomated
+
+        @CaseLevel: Integration
+        """
+
+    @stubbed()
+    @run_only_on('sat')
+    @tier2
+    def test_positive_remove_promoted_cv_version_from_default_env(self):
+        """Remove promoted content view version from Library environment
+
+        @id: a8649444-b063-4fb4-b932-a3fae7d4021d
+
+        @Steps:
+
+        1. Create a content view
+
+        2. Add a puppet module(s) to the content view
+
+        3. Publish the content view
+
+        4. Promote the content view version from Library -> DEV
+
+        5. remove the content view version from Library environment
+
+        @Assert:
+
+        1. Content view version exist only in DEV and not in Library
+
```

Comment:
empty lines between steps are not required. and its applicable to other stubs as well.